src/sass/modules/_m-schemaform.scss

```diff
@@ -231,3 +231,12 @@ label {
     padding-left: 25px !important;
   }
 }
+
+.usa-button.schemaform-file-remove-button {
+  width: auto;
```

Comment:
Nitpicky, but I'd like to see a bit of left padding too (@goldenmeanie to confirm, though).